.vault-config/productconstructionint.yaml

```diff
@@ -22,3 +33,22 @@ secrets:
       dataSource: tcp:maestro-int-server.database.windows.net,1433
       database: BuildAssetRegistry
       permissions: r
+
+  BotAccount-dotnet-bot-repo-PAT:
+    type: github-access-token
```

Comment:
This token gives INT the write permissions to every repo?

Reply:
The only scope I gave it it public_repo, which gives it Access to Public Repositories, so yes, it should

Reply:
I am not sure whether we shouldn't create a token for the `maestro-auth-test` org only. Or maybe for https://github.com/maestro-auth-test/dnceng-vmr and the repos that it contains only.

Reply:
Also, the VMR will have more remotes - AzDO and GitHub. But let's not complicate things now with this. It won't be needed for quite some time.